### DISCUSSION TITLE
Missing Highlight for Newly Added Line in Tic-Tac-Toe Tutorial

### DISCUSSION THREAD

**Discussion by jisunipark** [pmndrs/zustand#2927] [General]
I know this is a very minor issue, but I noticed there is a newly added line in the Tic-Tac-Toe tutorial that isn’t highlighted. It occurs in the second code block of the [“Taking Turns” section](https://zustand.docs.pmnd.rs/guides/tutorial-tic-tac-toe#taking-turns). The line `setXIsNext(!xIsNext)`, which is the last line of the `handleClick` function, is new compared to the previous code snippet, yet it isn’t highlighted.

**Comment by dai-shi** [comment]
Thanks. Feel free to send a PR!

https://github.com/pmndrs/zustand/blob/2e6d8813095c6a79ca208bae4c2cf5edc12049a1/docs/guides/tutorial-tic-tac-toe.md?plain=1#L798
